src/base/bittorrent/session.h

```diff
@@ -198,43 +204,43 @@ namespace BitTorrent
     {
         struct
         {
-            int hasIncomingConnections = 0;
-            int sentPayloadBytes = 0;
-            int recvPayloadBytes = 0;
-            int sentBytes = 0;
-            int recvBytes = 0;
-            int sentIPOverheadBytes = 0;
-            int recvIPOverheadBytes = 0;
-            int sentTrackerBytes = 0;
-            int recvTrackerBytes = 0;
-            int recvRedundantBytes = 0;
-            int recvFailedBytes = 0;
+            std::size_t hasIncomingConnections = 0;
```

Comment:
technically, indices into arrays are `std::ptrdiff_t`, but I think standard containers got this wrong a long time ago. Heh, in fact, I got this wrong in my `span<>` as well. I will try to fix that.

Reply:
The standard containers are not arrays.